docs/v3.x/concepts/queries.md

```diff
@@ -353,9 +353,41 @@ const result = await strapi
 const fields = result.toJSON();
 ```
 
+### Knex
+
+Documentation: [http://knexjs.org/#Builder](http://knexjs.org/#Builder)
+
+You can access the Knex instance with 
+
+```js
+const knex = strapi.connections.default;
+```
+
+You can then use Knex to build your own custom queries to the DB. You will lose all the functionalities of the model, 
+but this could come handy if you are building a more custom schema.
+Please note that if you are using the draft system, Strapi nullyfies all the Draft columns util they are published.
+
+**Example**
+
+```js
+
+const knex = strapi.connections.default;
+const result = await knex('restaurants')
+  .where('cities', 'berlin')
+  .whereNot('cities.published_at', null)
+  .join('chefs', 'restaurants.id', 'chefs.restaurant_id')
+  .select('*')
+  .fetch();
+
+return result;
+```
+
+**We strongly suggest to sanitize any strings before making queries to the DB**
+Never attempt to make a raw query with data coming straigh from the front-end.
```

Comment:
straigh => straight

Reply:
You could also mention the `strapi.connections.default.raw()` for those that do need to use the raw SQL queries.

Reply:
I preferred not to mention the .raw() method because:
- using the raw query the response is different between mysql and postgres (the first returns an array, the latter an object), making the guide more confusing
- raw queries are more susceptible to SQL-injection attacks

Reply:
That's fine, we will leave it for when we review the docs. I agree on both points made but we still need to mention it eventually. No problem though if you don't want to, this PR is good as is thanks :)